kumascript/macros/CSSRef.ejs

```diff
@@ -1250,6 +1253,9 @@ async function buildPropertylist(pages, title) {
           <summary><%=text['Colors']%></summary>
           <ol>
             <li><%-smartLink(`${cssURL}CSS_colors/Applying_color`, null, text['Applying_color_to_HTML_elements_using_CSS'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_colors/Color_values`, null, text['Color_values'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_colors/Relative_colors`, null, text['Relative_colors'], cssURL)%></li>
+            <li><%-smartLink(`${cssURL}CSS_colors/Using_color_wisely`, null, text['Using_color_wisely'], cssURL)%></li>
```

Comment:
Can you please check if this works? 🙏  (If so, we don't need the translations above.)

```suggestion
            <li><%-smartLink(`${cssURL}CSS_colors/Color_values`)%></li>
            <li><%-smartLink(`${cssURL}CSS_colors/Relative_colors`)%></li>
            <li><%-smartLink(`${cssURL}CSS_colors/Using_color_wisely`)%></li>
```